compute_sdk/globus_compute_sdk/sdk/login_manager/tokenstore.py

```diff
@@ -36,11 +36,25 @@ def invalidate_old_config() -> None:
 
 
 def _ensure_funcx_dir() -> pathlib.Path:
-    dirname = _home() / ".funcx"
+    legacy_dirname = _home() / ".funcx"
+    dirname = _home() / ".globus_compute"
+
     try:
-        os.makedirs(dirname)
+        if os.path.exists(legacy_dirname) and os.path.isdir(legacy_dirname):
+            # If the legacy dir (.funcx) already exists, create a symlink
```

Comment:
- Let's symlink from `.funcx` to `.globus_compute`, rather than vice-versa.
- Use the `pathlib.Path` class&nbsp;&mdash;&nbsp;no need to resort to the thin fs wrappers.  Can elide `try` and the redundant `exists()` calls.  Consider:
    ```python
    def _ensure_funcx_dir() -> pathlib.Path:
        legacy_dirname = _home() / ".funcx"
        dirname = _home() / ".globus_compute"

        if dirname.is_dir():
            pass  # work already done

        elif legacy_dirname.is_dir():
            legacy_dirname.replace(dirname)
            legacy_dirname.symlink_to(dirname, target_is_directory=True)

        else:
            dirname.mkdir(parents=True, exist_ok=True)

        return dirname
    ```

Reply:
In the description for [sc-22439](https://app.shortcut.com/funcx/story/22439/move-funcx-directory-to-globus-compute), the mention of renaming the legacy directory was crossed out, so I figured we were explicitly trying to avoid it. Looks like a markdown typo after reevaluating. This simplifies the change.